tests/test_cli.py

```diff
@@ -7,10 +7,23 @@
 import shutil
 import tempfile
 import textwrap
+import time
+import subprocess
+
+try:  # Python 3
```

Comment:
Nit: @t-henri Can we use the `six.moves` import for `urllib` instead (https://six.readthedocs.io/#module-six.moves.urllib.request).

Reply:
Done